zenoh/src/net/routing/interceptor/authorization.rs

```diff
@@ -199,10 +199,24 @@ impl PolicyEnforcer {
     ) -> ZResult<PolicyInformation> {
         let mut policy_rules: Vec<PolicyRule> = Vec::new();
         for config_rule in config_rule_set {
+            // config validation
+            if config_rule.interfaces.is_empty()
+                || config_rule.actions.is_empty()
+                || config_rule.flows.is_empty()
+                || config_rule.key_exprs.is_empty()
+            {
+                bail!("error from bad config");
```

Comment:
Error message is not specific enough.

```suggestion
                bail!("one or multiple ACL config lists are empty");
```

Reply:
I would separate the chained `||` checks in different ifs where the final log message is returned in case of error. In this way the user knows exactly what went wrong.